content/en/news/releases/1.17.x/announcing-1.17.1/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.17.1
+linktitle: 1.17.1
+subtitle: Patch Release
+description: Istio 1.17.1 patch release.
+publishdate: 2023-02-21
```

Comment:
I would recommend adding a time to the date, so something like: `2023-02-21T08:00:00-06:00` and changing the time so that the latest time is the 1.17, and earliest is the 1.15. Since these are part of different PRs it's not easy to tell if these will sort in the same order or not. You'll want to work with Daniel or check the other PRs to see what times are being used.

Reply:
I did T07:00:00-06:00 for 1.15, and T08:00:00-06:00 for 1.16 so anything after 08:00:00 should work.

Reply:
```suggestion
publishdate: 2023-02-21T09:00:00-06:00
```
Is what I tested with and this moved this to to top of the day's news items.